FunctionalEquivalence/subworkflows/F1Evaluation.wdl

```diff
@@ -299,7 +285,7 @@ def plot_aggregated_data_compact(data, stratifiers, dataset, tool1_label, tool2_
     fig.suptitle('Dataset: {}'.format(dataset) + ('' if not additional_label else ', {}'.format(additional_label)) + '\n' +
                 ('Signed ' if signed_difference else 'Absolute ') +
                 r'$F_1 = \frac{TP}{TP + \frac{1}{2} (FP + FN)}$ score differences for calls with $QUAL \geq q$' +
-                ' (# replicates: {})'.format(len(data[dataset])))
+                ' (# replicates: {})'.format(len(data.loc[data['dataset'] == dataset].replicate.unique())))
```

Comment:
This figure title in particular can be made more readable with some f-strings (see last comment).